<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetChannelDialog</class>
 <widget class="QDialog" name="SetChannelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="AddButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="DeleteButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>偏移量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LEOffSet">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>数据长度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LELength">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>通道配置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>器高8位，振动传感器低8位;...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="DelSetButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>清除配置</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>通道号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LEChnanelNum">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>数据类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="BoxDataType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>X寄存器</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y寄存器</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>指令电机位置</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>实际电机位置</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>指令位置</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>实际位置</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>通道变量</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>轴变量</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>系统变量</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>逻辑轴号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="BoxAxisNo">
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>22</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>X轴</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y轴</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z轴</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C轴</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="SetButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>配置</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>配置说明：电机实际角度值;振动传感</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
